tilelang/contrib/msvc.py

```
@@ -0,0 +1,286 @@
+    for lib_dir in TL_LIBS:
+        if not os.path.isdir(lib_dir):
+            continue
+        for lib_name in ("tvm.lib", "tvm_ffi.lib"):
+            lib_path = os.path.join(lib_dir, lib_name)
+            add_import_lib(lib_path)
+
+    try:
+        from tvm_ffi import libinfo as tvm_ffi_libinfo
+
+        add_import_lib(tvm_ffi_libinfo.find_windows_implib())
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)